csrc/ir/interface_nodes.h

```diff
@@ -400,12 +400,11 @@ class NVF_API TensorView : public Val {
 
   // Returns true if this tensor is circular buffered.
   bool isCircularBuffered() const {
-    return is_circular_buffered_;
+    return circular_buffer_stage_ > 1;
   }
 
   // Returns the depth of circular buffering if applicable.
   int64_t circularBufferDepth() const {
-    NVF_ERROR(is_circular_buffered_, toString(), "not circular buffered");
```

Comment:
This check is removed because I somehow feel it convenient to write code like:
```C++
auto depth = tv->circularBufferDepth();
if (depth > 1) {
  do_something_for_circular_buffering(depth);
}
```